cmd/aks-periscope/aks-periscope.go

```diff
@@ -26,10 +31,6 @@ func main() {
 		log.Fatalf("Failed to get the hostname on which AKS Periscope is running: %v", err)
 	}
 
-	if err := utils.CreateCRD(); err != nil {
```

Comment:
question: will we still write and apply the crd?

Reply:
💡 Thank you so much for this question: Not sure why this code existed in first place, like we use `daemonset` to deploy against cluster, why do we need `createCRD` against a node and create a `diagnostic.yaml` file which seems to be doing nothing? 

hence I removed this, and with new image published as test `docker.io/tatsat/misc-move-to-client-go` I see no difference in the output of this tool, so I wonder what was the use of this `createCRD`.

Reply:
Got it, thanks for the context

Reply:
Thanks heaps to you bro, I think it is one of the very good question and I've wondered why this code existed, given its now 2 years, I wonder if there is more context to add @JunSun17 , but it works fine without it. ❤️🙏☕️